xcodeproj/internal/target.bzl

```diff
@@ -862,6 +877,18 @@ def _passthrough_target(*, transitive_infos):
         ),
     )
 
+def _process_dependencies(*, transitive_infos):
+    return [
+        dependency
+        for dependencies in [
+            # We pass on the next level of dependencies if the previous target
+            # didn't create an Xcode target.
+            [info.target.id] if info.target else info.dependencies
+            for info in transitive_infos
+        ]
+        for dependency in dependencies
+    ]
```

Comment:
is this a test?

Reply:
I'll add a utility method to clean these up after I land my stack of PRs.